src/lib/services/apiResponseWrapper.ts

```diff
@@ -33,37 +33,35 @@ export type ApiResponseWrapperError<T> = {
     message: string;
 };
 
-export class ApiResponseWrapperUtil {
-    static fromErrorCode<T>(error: ApiResultStatus, message: string): ApiResponseWrapperError<T> {
-        return {
-            isSuccess: false,
-            result: undefined,
-            errorCode: error,
-            message: message,
-        };
-    }
-
-    static async fromResponse<T>(response: Response): Promise<ApiResponseWrapper<T>> {
-        const status = getStatus(response.status);
-        if (status === ApiResultStatus.SUCCESS) {
-            return {
-                isSuccess: true,
-                result: await response.json(),
-            };
-        } else {
-            return {
-                isSuccess: false,
-                result: await response.json(),
-                errorCode: status,
-                message: response.statusText,
-            };
-        }
-    }
+export function WrapErrorCode<T>(error: ApiResultStatus, message: string): ApiResponseWrapperError<T> {
```

Comment:
As already mentioned by Frederik, I'd write the function names lowercase. Apart from that, I like the function names a lot.